doc/api/repl.md

```diff
@@ -774,6 +774,14 @@ a `net.Server` and `net.Socket` instance, see:
 For an example of running a REPL instance over [`curl(1)`][], see:
 <https://gist.github.com/TooTallNate/2053342>.
 
+This example is intended purely for educational purposes to demonstrate how
```

Comment:
```suggestion
All examples are intended purely for educational purposes to demonstrate how
```

Reply:
Is there another example that poses a security risk? I don't think so

Reply:
I'm thinking more of a just-in-case scenario, if in the future one of them presents a risk, this could act as a catch-all. It's a nitpick, so feel free to ignore.

Reply:
Alright, my intention in this note is to highlight this specific example. If we mention all of them, users might think we are saying, "Don't use all of this in production."